lib/engine/game/g_18_india/game.rb

```diff
@@ -594,6 +594,19 @@ def legal_tile_rotation?(_entity, _hex, _tile)
           true
         end
 
+        # Sell Train to the Depot
+        def sell_train(operator, train, price)
+          @bank.spend(price, operator) if price.positive?
+          @depot.reclaim_train(train)
+        end
+
+        def corp_is_trainless(corporation)
```

Comment:
`corp_is_trainless` sounds like it would just return a boolean, could this be renamed something like `handle_trainless_corp`? Something more descriptive along the lines of `drop_price_for_trainless_corp` is longer but fine too, up to you.

Reply:
Thank you for the suggestion.  I also need to move the timing of the drop in price.  I have confirmed with Ken that this happens at the end of the company's operating turn (after sell/buy certs, not after sell/buy trains).  I think I need to hook into `def after_end_of_turn(operator); end` from Engine::Round::Operating

Reply:
I made the suggested change, and moved the timing of the price drop for trainless corp.